zebedee-content-tools/publishing-timings/Makefile

```diff
@@ -0,0 +1,26 @@
+APP=publish-timings
+
+ENV?=sandbox
+
+GOOS?=$(shell go env GOOS)
+GOARCH?=$(shell go env GOARCH)
+
+BUILD=build
+BUILD_ARCH=$(BUILD)/$(GOOS)-$(GOARCH)
+
+.PHONY: build
+build:
+	go build -o $(BUILD_ARCH)/$(APP)
+
+.PHONY: build-aws
+build-aws:
+	make CGO_ENABLED=0 GOOS=linux GOARCH=amd64 build
+
+.PHONY: deploy
+deploy: build-aws
+	dp scp $(ENV) publishing_mount 1  -- $(BUILD)/linux-amd64/$(APP) $(APP);
```

Comment:
```suggestion
	dp scp $(ENV) publishing_mount 1  -- $(BUILD_ARCH)/$(APP) $(APP)
```
no semicolon needed

Reply:
Disagree for the env var. For deployment we want the AWS specific build. If this is run on an ARM based mac, it will try to upload the wrong binary with the env vrs as they will be set to the local arch not the target one. There is argument to say we should abstract the target arch too but probably not worth it for this script as we know we only use amd64 for our envs right now. Agree for the semicolon though.

Reply:
Aside: `BUILD_ARCH` is a *make* var (if you use `make BUILD_VAR=xyz`)

My suggestion was also concerned with the dependency (line 20) on `build-aws` which would (very likely) build the different *local* target (`darwin-amd64` typically) -
and not refresh the `linux-amd64` one - before uploading the latter, assuming that you've (ever) run the build with the right make vars (`GOOS` and `GOARCH`) - inconsistent/hidden effects.